lighthouse/data-fixes/data_fix_and_save.py

```diff
@@ -0,0 +1,15 @@
+# get the root_sample_ids, fix them, write these to a CSV to be used with the 'write_data' script (which inserts the fixed IDs into the DBs)
```

Comment:
I would group all files related with the same data fix together, so we can add other data fixes in future if needed but keeping things separate, what do you think on creating a subfolder dpl-048 and move all files inside there?

Reply:
Also, for future reference, it will be very useful to have a README.md file in that subfolder with the description of this data fix and how to run it.